js/counter.js

```diff
@@ -0,0 +1,9 @@
+setInterval(setCounter, 1000
```

Comment:
if you use something, it's best practice to write yourself after it (to make sure it exists) :)